compiler/luci-interpreter/src/kernels/Squeeze.test.cpp

```diff
@@ -26,58 +26,51 @@ namespace
 
 using namespace testing;
 
-TEST(SqueezeTest, Float)
+template <typename T>
+void Check(std::initializer_list<int32_t> input_shape, std::initializer_list<int32_t> output_shape,
+           std::initializer_list<float> input_data, std::initializer_list<float> output_data,
+           DataType element_type, int8_t squeeze_dims_count, std::vector<int32_t> squeeze_dims)
 {
-  std::vector<float> input_data{1.0,  2.0,  3.0,  4.0,  5.0,  6.0,  7.0,  8.0,
-                                9.0,  10.0, 11.0, 12.0, 13.0, 14.0, 15.0, 16.0,
-                                17.0, 18.0, 19.0, 20.0, 21.0, 22.0, 23.0, 24.0};
-  Shape input_shape = {1, 24, 1};
-  Tensor input_tensor = makeInputTensor<DataType::FLOAT32>(input_shape, input_data);
-  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+  std::pair<float, int32_t> quant_param = quantizationParams<T>(-127.f, 128.f);
```

Comment:
Do we have to test with quantizationParams? Squeeze does not change quantizationParams.

A similar case (`Transpose.test.cpp`), which does not change data, does not use quantizationParams for testing uint8 case.

Reply:
okay. I will remove this and update the test cases.